buildenv/jenkins/README.md

```diff
@@ -36,34 +36,31 @@ This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkin
     - AIX PPC (aix)
     - Windows 64 bits (win)
     - Windows 32 bits (win32) - supported on JDK8 only
-- Current supported Java verisons are Java8 and Java9
 - OpenJ9 committers can request builds by commenting in a pull request
-    - Format: `Jenkins <build type> <level> <platform(s)> <java version(s)>`
-    - Build Types: compile,test
-    - Levels: sanity,extended (only if Build Type is test)
-    - Platforms: xlinux,zlinux,plinux,aix,win,win32
-    - Java Versions: jdk8,jdk9,jdk10
-- Note: You can use keyword `all` for level, platform or version
+    - Format: `Jenkins <build type> <level> <platform>[,<platform>,...,<platform>] jdk<version>[,jdk<version>,...,jdk<version>]`
+    - `<build type>` is compile | test
+    - `<level>` is sanity | extended (required only for "test" `<build type>`)
+    - `<platform>` is aix | xlinux | xlinuxlargeheap | zlinux | plinux | win | win32
+    - `<version>` is the number of the supported release, e.g. 8 | 11 | n 
+- Note: You can use keyword `all` for platform
 
 ###### Examples
-- Request a Compile-only build on all platforms and all java versions by commenting in a PR
-    - `Jenkins compile`
-    - `Jenkins compile all all`
-- Request a Sanity build on zLinux all java versions
-    - `Jenkins test sanity zlinux`
-    - `Jenkins test sanity zlinux all`
-- Request an Extended build on pLinux java8 only
+- Request a Compile-only build on all platforms and multiple versions by commenting in a PR
+    - `Jenkins compile all jdk8,jdk11`
+- Request a Sanity build on zLinux and multiple versions
+    - `Jenkins test sanity zlinux jdk8,jdk11`
+- Request an Extended build on pLinux for a single version
     - `Jenkins test extended plinux jdk8`
-- Request a Sanity build on z,p Linux on java 8,9
+- Request a Sanity build on z,p Linux for multiple versions
     - `Jenkins test sanity zlinux,plinux jdk8,jdk9`
-- Request all tests on all platforms all java versions
-    - `Jenkins test all`
-    - `Jenkins test all all all`
+- Request Sanity tests on all platforms and multiple versions
+    - `Jenkins test sanity all jdk8,jdk11`
+
+You can also request a Pull Request build from the Eclipse OpenJ9 repository - [openj9](https://github.com/eclipse/openj9) - or the Extensions OpenJDK\* for Eclipse OpenJ9 repositories:
 
-You can also request a Pull Request build from the extensions repos or openj9-omr
-- [openj9-omr](https://github.com/eclipse/openj9-omr)
 - [openj9-openjdk-jdk8](https://github.com/ibmruntimes/openj9-openjdk-jdk8)
 - [openj9-openjdk-jdk9](https://github.com/ibmruntimes/openj9-openjdk-jdk9)
+- etc
```

Comment:
How about adding openj9-openjdk-jdk and link openj9-openjdk-jdk<version> and non-hidden url for user to copy and construct?